lib/features/give/dialogs/give_to_family_goal_loading_dialog.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app/features/give/bloc/bloc.dart';
+import 'package:givt_app/l10n/l10n.dart';
+import 'package:givt_app/utils/app_theme.dart';
+import 'package:go_router/go_router.dart';
+
+class GiveToFamilyGoalLoadingDialog extends StatelessWidget {
+  const GiveToFamilyGoalLoadingDialog({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.sizeOf(context);
+    return BlocListener<GiveBloc, GiveState>(
+      listener: (context, state) {
+        if (state.status == GiveStatus.processed ||
+            state.status == GiveStatus.error) {
+          context.pop();
+        }
+      },
+      child: SimpleDialog(
+        backgroundColor: AppTheme.givtGraycece,
+        children: [
+          SizedBox.square(
+            dimension: size.width * 0.5,
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                const CircularProgressIndicator.adaptive(),
```

Comment:
Do we want to show native indicator or same progress indicator on every platform (material)? :)

Reply:
Idk, let's ask Elmar in Slack :)